dockerFiles/marklogic-deps-centos:base

```diff
@@ -13,7 +13,7 @@ LABEL "com.marklogic.maintainer"="docker@marklogic.com"
 
 RUN echo "NETWORKING=yes" > /etc/sysconfig/network \
     && yum -y update \
-    && yum -y install gdb.x86_64 glibc.i686 initscripts redhat-lsb-core.x86_64 tzdata xz-0:5.2.2-2.el7_9.x86_64 \
+    && yum -y install gdb.x86_64 glibc.i686 libstdc++.i686 libgcc.i686 initscripts redhat-lsb-core.x86_64 tzdata xz-0:5.2.2-2.el7_9.x86_64 \
```

Comment:
@rwinieski why did these get removed in your feature branch?

Reply:
That's a good question actually. I do not know.